istio-telemetry/tracing/values.yaml

```diff
@@ -0,0 +1,77 @@
+#
+# addon tracing configuration
+#
+tracing:
+  enabled: false
+
+  provider: zipkin
```

Comment:
Is the default provider being changed for a particular reason?

Reply:
I have never even used tracing, so not an expert on it by any means, but why do we have tracing.provider=jaeger and global.proxy.tracer=zipkins? what is even the difference? We have so many different tracing configs..

Reply:
I know it can be confusing. The `tracing.provider` refers to the backend tracing system. The `global.proxy.tracer` refers to the tracing client being used by the proxy - i.e. could be lightstep, zipkin, etc.

Jaeger can consume the zipkin data format, hence the tracer can be zipkin, but the tracing.provider can be jaeger.

Reply:
Thanks for clearing that up. If that is the case then it looks like this was not intentionally changed, we should keep in line with current master (Jaeger)?

Reply:
+1

Reply:
I changed this when I was testing with zipkin locally and forgot to revert it back, change back to jaeger now.